packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts

```diff
@@ -297,7 +352,7 @@ export class OpenId4VciHolderService {
 
     this.logger.debug('Requested OpenId4VCI Access Token.')
 
-    return accessTokenResponse.successBody
+    return { ...accessTokenResponse.successBody, dPoPJwk }
```

Comment:
This is infortunate, but i don't think we should mix it. Ideally we return a new object with keys: {response: {body,headers,code}, dpop: {jwk}}


Then It's more extendable. 

Maybe deprecate and add a new one?  Or we keep it like this for now and upgrade when moving to 0.6


But still would like to have dpop.jwk so we can extend dpop values as well.

Reply:
I would leave it as is for now. Maybe change with 0.6 then.

> But still would like to have dpop.jwk so we can extend dpop values as well. 
On the client side you only receive the dpop not really things that can be extended IMO

Reply:
Until we maybe want to return the dpop that was sent, or the htu, or anything.. I think we've had a few times now where not adding an extra layer of nesting resulted in using having to make breaking changes. Rather take that into account now